src/prefect/runtime/flow_run.py

```diff
@@ -248,6 +250,13 @@ def get_parent_deployment_id() -> Dict[str, Any]:
     return parent_flow_run.deployment_id if parent_flow_run else None
 
 
+def get_flow_run_url() -> Optional[str]:
```

Comment:
Should we differentiate this as the UI URL (as opposed to the API URL)?

Reply:
hmm yeah i thought about making this `ui_url`

Reply:
Maybe we should provide both?  `.api_url` and `.ui_url`?  Both seem useful in different contexts

Reply:
added in [fdef7bf](https://github.com/PrefectHQ/prefect/pull/11686/commits/fdef7bfcb13820c633642de0285cba7cf5149006)